### ISSUE TITLE
UI/UX: a different take

### ISSUE THREAD

**Issue by ricardoduplos** [developmentseed/worldpop#43]
@anandthakker Here's a quick sketch proposal on how tools and results could be merged and consolidated into one side panel.

![scan](https://cloud.githubusercontent.com/assets/1016701/7685346/9e05dbf8-fd85-11e4-827b-060888a71e0a.jpg)

In this version, the About text and footer notes would be presented in a modal dialog (via header link).

**Comment by anandthakker** [comment]
@ricardomestre :100: I really like this.  Feels like this is really how this tool _wants_ to be.

cc @smit1678

**Comment by smit1678** [comment]
@ricardomestre Yeah, dig the direction. Really like the save and delete your previous measurement functionality.
